fenix/app/src/main/java/org/mozilla/fenix/components/metrics/GrowthActivatedUserWorker.kt

```diff
@@ -0,0 +1,74 @@
+package org.mozilla.fenix.components.metrics
+
+import android.content.Context
+import androidx.work.ExistingWorkPolicy
+import androidx.work.OneTimeWorkRequest
+import androidx.work.WorkManager
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import mozilla.components.support.utils.ext.getPackageInfoCompat
+import org.mozilla.fenix.ext.metrics
+import org.mozilla.fenix.ext.settings
+import java.util.concurrent.TimeUnit
+
+/**
+ * Worker that will send the User Activated event at the end of the first week.
+ */
+class GrowthActivatedUserWorker(
```

Comment:
Maybe `GrowthDataWorker` would keep this slightly more decoupled, allowing us to add more events to it over time?

Reply:
gotcha, I'll change the name